external/avro/src/test/scala/org/apache/spark/sql/avro/AvroSuite.scala

```diff
@@ -809,4 +810,16 @@ class AvroSuite extends SparkFunSuite {
       assert(readDf.collect().sameElements(writeDf.collect()))
     }
   }
+
+  test("SPARK-24805: reading files without .avro extension") {
```

Comment:
Nit: Can we create a temp path and copy the original `episodes.avro` to the path? So that we don't need to have two duplicated resource file.

Reply:
Does it just introduce unnesseccary dependency here and overcomplicate the test? I can create small (with just one row) avro file without `.avro` extension especially for the test.